design-documents/service-isolation/inter-service-communication.md

```diff
@@ -0,0 +1,294 @@
+### Inter-service communications
+This documents describes how exactly services will be communicating in a distributed setup.
+#### Service declaration
+Services will be declared in Api modules using interfaces. These modules will be installed on all nodes.
```

Comment:
Do you mean installed on all nodes of particular service? For instance, ServiceBApi module will be installed only on ServiceB? Please clarify.